### PR TITLE
Adds suppermatter

### PR DESCRIPTION
**Pull request by PJB3005**
:cl:
- add: Added suppermatter. Goes great with a glass of syndicate bomb!

image

Original item from /vg/, but I made the sprites myself by cutting up the TG's SM sprite because the /vg/ one is pretty awful.

### CODE REVIEW

**Comment by PJB3005** [comment]
I spent like 20 minutes debugging why the microwave wouldn't work just to make this.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 1]
```diff
@@ -617,3 +617,61 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 12
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 12
+  - type: Food
+    transferAmount: 3
+    trash: ""
```
Comment (PJB3005) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:673]: Yaml linter is broken and says this line is bad but it's totally fine.

**Comment by Mirino97** [comment]
I read this as you adding the SM engine.

DAM YOU PJB, YA'VE BLUE BALL'D ME!!

**Review by moonheart08 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml [hunk 2]
```diff
@@ -617,3 +617,61 @@
...
+          Quantity: 48
+  - type: Food
+    transferAmount: 12
+    trash: FoodPlate
+  - type: Item
+    size: 40
+  - type: PointLight
+    color: "#FFFF00"
+    radius: 2
+    energy: 1.4
```
[the remark this answers is not in the document]
Comment (mirrorcult) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:653]: consider that its funny

[the remark this answers is not in the document]
Comment (moonheart08) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:653]: > consider that its funny

rn due to robust being weird, exceeding the light limit causes nasty artifacting that's really hard on people's eyes.

[the remark this answers is not in the document]
Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml:653]: pjb issue.

**Pull request merged**